components/todo/utils/TodoActions.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <div class="action-buttons">
+    <button
+      v-if="!isTodoCompleted"
+      class="todo-action"
+      @click.prevent="completeTodo"
+    >
+      {{ $t('complete') }}
+    </button>
+    <button
+      v-if="!isTodoCompleted"
+      class="todo-action"
+      @click.prevent="editTodo"
+    >
+      {{ $t('edit') }}
+    </button>
```

Comment:
Could be a button group as we're using the same state for `v-if` multiple times.

Reply:
resolved